releases/ALUnit-v1.0.lsp

```diff
@@ -304,7 +304,7 @@ SOFTWARE.
  | *ALU:testSuites* - assoc list of test suites and a list of their tests
  | *ALU:testsRun* - counter for total tests run in the current batch
  | *ALU:failMessages* - list of messages to print after testing
- | *ALU:startTime* - date number for start of testing
+ | *ALU:startTime* - milliseconds since boot time for start of testing
```

Comment:
This looks like good stuff, but I don't think it should go in this release file. Could you create a new release file for these changes? v1.1.0 sounds right to me as this is an important change, but shouldn't break anything, right?